operator/cmd/metrics/adapter/adapter.go

```diff
@@ -0,0 +1,80 @@
+package adapter
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	kmprovider "github.com/keptn/lifecycle-toolkit/operator/cmd/metrics/adapter/provider"
+	"k8s.io/apiserver/pkg/server/options"
+	"k8s.io/component-base/logs"
+	"k8s.io/klog/v2"
+	basecmd "sigs.k8s.io/custom-metrics-apiserver/pkg/cmd"
+	"sigs.k8s.io/custom-metrics-apiserver/pkg/provider"
+)
+
+const (
+	FlagPort                 = "adapter-port"
+	FlagCertificateDirectory = "adapter-certs-dir"
+)
+
+var (
```

Comment:
do these variables need to be global? Cannot we make them local  in RunAdapter() func ?